charts/refractr/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: refractr
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
```

Comment:
this is just my preference, but I'd get rid of the comments. Maybe leaving only the part which explains the behavior of a certain option.

Reply:
Yeah that was just default boilerplate that isn't so helpful now -- removed.